src/test/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClientTest.java

```diff
@@ -125,6 +125,27 @@ public void connectsSecurely() {
     assertEquals(HttpHost.create("http://localhost:9200"), preemptiveAuthTargetHosts.iterator().next());
   }
 
+  @Test
+  public void compressedConnectsSecurely() {
```

Comment:
this mimics all the other tests - let's at least check that the compression was set